lessons/LargeDatasetPython/lesson.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title:
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# Large Datasets in Python (Aug 25, 2016)
+
+ - **Authors**: Wei Zhan
+ - **Research field**: Immunology
+ - **Lesson topic**: Processing gDNA chip results and single cell PCR results; finding shared motifs.
+ - **Lesson content URL**: <https://github.com/gattacagithub/studyGroup/tree/gh-pages/lessons/LargeDatasetPython>
+
+This lesson contains two parts: 1st, accessing and storing data from personal genomic DNA sequencing results using the pandas.DataFrame structure; 2nd, finding shared motifs in single-cell PCR sequencing results.
+
+To follow along, visit [the IPython notebook](https://github.com/gattacagithub/studyGroup/blob/gh-pages/lessons/LargeDatasetPython/LargeDatasetsPython.ipynb).
+
+The [generic_gdna.txt](https://github.com/gattacagithub/studyGroup/blob/gh-pages/lessons/LargeDatasetPython/generic_gdna.txt) file contains a sample personal gDNA chip sequencing output.
```

Comment:
link to: `https://github.com/uoftcoders/studyGroup/blob/gh-pages/lessons/LargeDatasetPython/generic_gdna.txt`